prow/plugins.yaml

```diff
@@ -43,6 +45,7 @@ lgtm:
   - istio/test-infra
   - istio/proxy
   - istio-releases/pipeline
+  - istio-ecosystem/authservice
   trusted_team_for_sticky_lgtm: "Istio Hackers"
 
```

Comment:
Same comment here you might want to create another section

Reply:
fixed